relay-server/src/envelope.rs

```diff
@@ -771,6 +783,17 @@ impl Item {
         self.headers.replay_combined_payload = combined_payload;
     }
 
+    /// Returns the payload's replay video events.
+    #[cfg(feature = "processing")]
+    pub fn replay_video_events(&self) -> Option<(Vec<u8>, Vec<u8>)> {
+        self.headers.replay_video_events.clone()
```

Comment:
clone here seems a bit too implicit, I get that we need an owned value to make a Bytes object, but maybe defer cloning to the caller so they know it's potentially expensive.  
 
we could also look into something like take_video_events if we only need to use it once and the payload is large